docs/9.x/changelog.md

```diff
@@ -882,6 +899,16 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 6.1 Releases
 
+### 6.1.49 LTS (August 4, 2021)
+
+#### Improvements
+* Rollback of dns-app, monitoring-app, gravity-site are now re-entrant ([#2589](https://github.com/gravitational/gravity/pull/2589), [rigging#87](https://github.com/gravitational/rigging/pull/87)).
+* Update of dns-app, monitoring-app, gravity-site are now re-entrant ([#2589](https://github.com/gravitational/gravity/pull/2589), [rigging#86](https://github.com/gravitational/rigging/pull/86)).
+
+#### Bugfixes
+* Add missing retry loop when updating taints ([#2581](https://github.com/gravitational/gravity/pull/2581)).
+* Rework the installation and upgrade of the logging-app ([#2589](https://github.com/gravitational/gravity/pull/2589), [logging-app#81](https://github.com/gravitational/logging-app/pull/81)).
```

Comment:
In other releases, this was:

```suggestion
#### Internal Changes
* Change ownership of logging-app data ([#2589](https://github.com/gravitational/gravity/pull/2589), [logging-app#79](https://github.com/gravitational/logging-app/pull/79)).
```
I'd recommend at least using the same description -- to make tracking relation easier.  "Internal Changes" is take it or leave it. Also https://github.com/gravitational/gravity/pull/2447 was previously grouped with this -- did that need to be ported too?

Add:
```
 * Fix a logging-app panic ([#2589](https://github.com/gravitational/gravity/pull/2589), [logging-app#79](https://github.com/gravitational/logging-app/pull/79)).
```

Reply:
unfortunately it's not the same patch, I had to back some of it out in 6.1 because it's incompatible with either kubernetes or gravity. So that isn't actually the case.

Reply:
Understood -- ownership didn't change will data still be abandoned?

Reply:
So I decided to test this out more closely and am glad I did, it turns out this would prevent an upgrade to 7.0, as this would create the new directories as root and then on 7.0 would try to access them as the planet user and fail. So I'll have to re-release 6.1 backing out the directory changes and it won't abandon any data.

Reply:
Good catch.  Forward upgrades is something I've had on my robotest radar for a long time.  https://github.com/gravitational/robotest/issues/186

I *think* I added in support for this in https://github.com/gravitational/robotest/pull/259, but I've never added this scenario to the configs gravity side.  It may be worth implementing if we're planning to do some big upgrade chains (e.g. 5.5 -> 9.0) in the future.